data/comid/README.md

```diff
@@ -0,0 +1,83 @@
+
+# CoMID Template Format
+
+## 1 Introduction
+
+**CoMID (Concise Model Identifier)**,  is a data model and serialization format (in JSON) for capturing **reference values** and **verification keys** that can be used in remote attestation and other trust-verification scenarios. By standardizing how measurements are captured and shared, CoMID facilitates **interoperability**, **integrity**, and **traceability** across various systems and vendors.
+
+## 2 Template Structure
+
+A CoMID template is a JSON document composed of **top-level fields** and a **triples** object. The **top-level fields** provide overall identification, language, and authorship, while the **triples** object contains domain-specific data (e.g., reference values, attester keys).
+
+```
+{
+  "lang": "<language-region>",
+  "tag-identity": { ... },
+  "entities": [ ... ],
+  "triples": {
+    "reference-values": [ ... ],
+    "attester-verification-keys": [ ... ]
+    ...
+  }
+}
+``` 
+
+### 2.1 Top-Level Fields
+
+-   **lang** (`String`): Defines the language or locale (e.g., `"en-GB"`).
+-   **tag-identity** (`Object`): Uniquely identifies this CoMID document via an ID (often a UUID) and includes a version number.
+-   **entities** (`Array`): Lists the entities (organizations, individuals, etc.) contributing to or maintaining the document, along with their roles.
+
+### 2.2 Triples
+
+-   **reference-values**: One or more **reference-value** objects, each containing an **environment** and one or more **measurements**.
+-   **attester-verification-keys**: One or more **attester-verification-key** objects, each containing an **environment** and an array of **verification-keys**.
+
+
+## 3 Components
+
+### 3.1 Environment
+
+An **environment** captures the context of a measurement or verification key:
+
+-   **class**: Vendor, model, and possibly an ID (`type` + `value`).
+-   **instance** (`optional`): For distinguishing multiple instances of the same environment (e.g., using `ueid` or `uuid`).
+-   **layer** and **index** (`optional`): For layered environments (e.g., DICE layers in multi-stage boot processes).
+
+### 3.2 Measurements
+
+Each measurement has two crucial subfields:
+
+-   **key**: Identifies the measurement, including possible fields like `label`, `version`, and `signer-id`.
```

Comment:
These don’t match the CoMID spec.

Reply:
@deeglaze could you be more  specfic as i got it in most of the templates like
```

            "key": {
              "type": "psa.refval-id",
              "value": {
                "label": "PRoT",
                "version": "1.3.5",
                "signer-id": "rLsRx+TaIXIFUjzkzhokWuGiOa48a/2eeHH35di66Gs="
              }
            }
```